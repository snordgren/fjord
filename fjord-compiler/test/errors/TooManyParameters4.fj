module test.errors.TooManyParameters4

record Zero
  zero : Int

implicit
zeroInstance : Zero
zeroInstance = 
  Zero 0

myFun : implicit Zero -> Int
myFun ev x =
  0
