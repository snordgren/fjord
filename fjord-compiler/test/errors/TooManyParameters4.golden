TooManyParameters4.fj:12:10:
   |
12 | myFun ev x =
   |          ^^^^
too many parameters, expected 1
